README.md

```diff
@@ -17,6 +17,13 @@
 [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-r/depshield.svg)](https://depshield.github.io)
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-r.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-r%22)
 
+> **Huzzah!** R is now part of Nexus Repository Manager. Version 3.next includes the R plugin by default. The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-r](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-r)
+
+> **Filing issues:** If using 3.next or later please file any issues at https://issues.sonatype.org/.
+
+> **Upgrading to 3.next:** If you are an existing R user upgrading to 3.next you will not be able to install the
```

Comment:
We should probably state the 'to' version. i.e. if you using a version prior to 3.20 and upgrade to a newer version you will not be able to install the community plugin. No other changes are required, and your existing data will remain intact.

Reply:
@jlstephens89 
let's imagine, code base was moved 3.29 for example. 
Maybe we should say
```
If you using a version prior to 3.20 - 3.28 and upgrade to a newer version you will not be able to install the community plugin. No other changes are required, and your existing data will remain intact.
```
I don't know version, and that's why I put 3.next

Reply:
This is true for all versions 3.20 and later. The range would just be confusing. 

Does it matter where the source lived for 3.20 to 3.28? As long as we're clear it is in a different place now I'm happy.

Reply:
I thought about it and it doesn't matter where the source code lived

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-r/pull/139/commits/15e957ddc6f9601b900eb54b344bde389f98792e